ios-xcframework/Podfile

```diff
@@ -51,21 +36,25 @@ DEPENDENCIES = %w[
   RNCMaskedView
   RNCClipboard
   RNFastImage
-  React-Codegen
   React-jsc
-  React-hermes
 ].freeze
 
 def gutenberg_dependencies
   podspec_prefix = '..'
 
-  # FBReactNativeSpec needs special treatment because of react-native-codegen code generation
-  pod 'FBReactNativeSpec',
-      podspec: "#{podspec_prefix}/third-party-podspecs/FBReactNativeSpec/FBReactNativeSpec.podspec.json"
+  computed_dependencies = DEPENDENCIES.dup
+
+  # Use a custom RNReanimated version while we coordinate a fix upstream
+  computed_dependencies.delete('RNReanimated')
+
+  computed_dependencies.delete('React-jsc') unless HERMES_ENABLED
 
-  DEPENDENCIES.each do |pod_name|
+  # Use a custom RNReanimated version while we coordinate a fix upstream
+  computed_dependencies.each do |pod_name|
     pod pod_name, podspec: "#{podspec_prefix}/third-party-podspecs/#{pod_name}.podspec.json"
   end
+
+  pod 'RNReanimated', git: 'https://github.com/wordpress-mobile/react-native-reanimated', branch: 'mokagio/fix-custom-node_modules-bypass-multiple-versions-check-2.17.0'
```

Comment:
Does it make sense to merge the fixes into [the `wp-fork` branch](https://github.com/wordpress-mobile/react-native-reanimated/pull/3) to follow our existing naming pattern or should we instead focus on dropping the fork altogether via an upstream merge?

Reply:
If I'm not wrong, the changes introduced in the branch `mokagio/fix-custom-node_modules-bypass-multiple-versions-check-2.17.0` are legit fixes that could be incorporated in the upstream repository. In any case, and in the meantime, I agree we could merge them into a `wp-fork` branch following the naming pattern.

My idea is to keep the previous forked branch, in case we need it before merging the upgrade, and create a new one for these changes. The updates I'm thinking to do are the following:
- Rename `wp-fork` branch to `wp-fork-2.9.1`.
- Create a new branch `wp-fork-2.17.0` and incorporate the fixes in the `podspec` file.

Reply:
This approach makes sense to me. I agree we should try to merge the fixes upstream as well.

Reply:
I finally created [this PR](https://github.com/wordpress-mobile/react-native-reanimated/pull/26), with the naming pattern we follow in forks, to include the workaround for Reanimated `2.17.0`. The pod reference has been updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5924/commits/630b73757c9a4139d6da33b7590ad06c4cc2916b.